bemoji

```diff
@@ -229,9 +229,37 @@ add_to_recent() {
 # Set default clipboard util
 _clipper() {
     if [ -n "$BEMOJI_CLIP_CMD" ]; then
-        # shellcheck disable=SC2068
-        ${BEMOJI_CLIP_CMD[@]}
-    elif [ -n "$WAYLAND_DISPLAY" ] && command -v wl-copy >/dev/null 2>&1; then
+        read -r -a cmd_array <<< "$BEMOJI_CLIP_CMD"
+        
+        if [ "${#cmd_array[@]}" -eq 1 ]; then
+            case "${cmd_array[0]}" in
+                wl-copy)
+                    if command -v wl-copy >/dev/null 2>&1; then
+                        wl-copy
+                        return
+                    fi
+                    ;;
+                xclip)
+                    if command -v xclip >/dev/null 2>&1; then
+                        xclip -selection clipboard
+                        return
+                    fi
+                    ;;
+                xsel)
+                    if command -v xsel >/dev/null 2>&1; then
+                        xsel -b
+                        return
+                    fi
+                    ;;
+            esac
+        else
+            # shellcheck disable=SC2068
+            ${BEMOJI_CLIP_CMD[@]}
+            return
+        fi
+    fi
+
+    if [ -n "$WAYLAND_DISPLAY" ] && command -v wl-copy >/dev/null 2>&1; then
```

Comment:
If we don't have a match in one of the above branches this still gets executed. This feels a little iffy to me,
as before there was an absolute guard from this being executed by running the [ -n "$BEMOJI_CLIP_CMD" ] conditional and instantly having a return at its branch end.
With the above code there are cases where we have a BEMOJI_CLIP_CMD set but still end up in this 'default-case' code below which seems muddy to me (e.g. when point 4 above occurs).